pb/event.pb.go

```diff
@@ -117,7 +117,9 @@ type ReviewEvent struct {
 	// Merge test branch where the PullRequest was merged.
 	Merge ReferencePointer `protobuf:"bytes,9,opt,name=merge" json:"merge"`
 	// Configuration contains any configuration related to specific analyzer
-	Configuration  google_protobuf2.Struct `protobuf:"bytes,10,opt,name=configuration" json:"configuration"`
+	Configuration google_protobuf2.Struct `protobuf:"bytes,10,opt,name=configuration" json:"configuration"`
+	// Number is internal provider id of review scoped by repository
+	Number         uint32 `protobuf:"varint,11,opt,name=number,proto3" json:"number,omitempty"`
```

Comment:
Maybe something more descriptive than just `Number`?

Reply:
any suggestions?
On github they call it `number`.
On gitlab `iid`.